test_unstructured/staging/test_weaviate.py

```diff
@@ -17,9 +17,10 @@
 )
 
 is_in_docker = os.path.exists("/.dockerenv")
+skip_outside_ci = os.getenv("CI", "").lower() in {"", "false", "f", "0"}
```

Comment:
This is a predicate (resolves to True or False) but is named like a function.

I'd say something like "is_in_ci" would be a better name, flip the logic to "not in" and then use `not is_in_ci` in the `skipif` expression.